html/janus.js

```diff
@@ -988,7 +990,20 @@ function Janus(gatewayCallbacks) {
 				ws.close();
 				ws = null;
 			} else {
-				navigator.sendBeacon(server + "/" + sessionId, JSON.stringify(request));
+        var uri = server + "/" + sessionId;
+        if (unloadParams) {
+          Object.keys(unloadParams)
+            .filter(function(key) { return unloadParams[key] != null; }) // Ignore null or undefined params
+            .forEach(function(key) {
+              // If there are more than 1 param, we need to use '&' instead of '?' after first param
+              if (uri.search(/\?.*=.*/) >= 0) {
+                uri += '&' + key + '=' + unloadParams[key];
+              } else {
+                uri += '?' + key + '=' + unloadParams[key];
```

Comment:
Shouldn't there be an escaping of those values? Or will `sendBeacon` do that itself?

Reply:
I will add an escaping for those values using `encodeURIComponent` in case someone uses `&` in a param.